sphinx-docs/Lateral-Movement-Guide.md

```diff
@@ -0,0 +1,92 @@
+Windows Lateral Movement Guide
+======================
+
+Using CALDERA's lateral movement and execution abilities, you can perform and test an adversary's capability to move 
+laterally within your network. This guide will walk you through some of the necessary setup steps to get started with 
+testing lateral movement in a Windows environment.  
+
+## Setup
+
+### Firewall Exceptions and Enabling File and Printer Sharing
+
+The firewall of the target host should not be blocking UDP ports 137 and 138 and TCP ports 139 and 445. The firewall
+should also allow inbound file and printer sharing. 
+
+```
+netsh advfirewall firewall set rule group="File and Printer Sharing" new enable=Yes
+```
+
+### User with Administrative Privileges
+
+This guide will assume a user *with administrative privileges to the target host* has been compromised and that a CALDERA
+agent has been spawned with this user's privileges. Scenarios where (1) the user has administrative privileges but is 
+not a domain account and (2) the user has administrative privileges and is a domain account are addressed in the 
+following sub sections.
+
+#### Non-Domain Account
+If the user with administrative privileges is *not* a domain account, remote UAC must be disabled on the target host by
+updating the following value in the registry. The command below will overwrite the existing registry value or add the
+vlaue if it doesn't already exist.
+```
+reg add HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System /v LocalAccountTokenFilterPolicy /t REG_DWORD /d 1 /f
```

Comment:
is this required to spawn a remote agent at all -- or only required to spawn an elevated remote agent?  If it only applies to service execution, then make a note of that (e.g. "because the techniques in this guide correspond to service execution, then remote UAC must be disabled when using a local administrator account on the the target machine")